src/app/[cityEnglishName]/[id]/page.tsx

```diff
@@ -0,0 +1,14 @@
+type Params = {
```

Comment:
I think it is CitiesProps + ReactElement for return type😅

Reply:
Pardon ))